types/src/extended_header.rs

```diff
@@ -28,6 +28,13 @@ pub struct ExtendedHeader {
     pub dah: DataAvailabilityHeader,
 }
 
+use core::fmt::{Display, Formatter};
+impl Display for ExtendedHeader {
```

Comment:
I found those useful, avoids printing entire Header in logs, not sure if we want this merged

Reply:
I think it's ok